src/test/java/com/coherent/task40/BaseTest.java

```diff
@@ -0,0 +1,16 @@
+package com.coherent.task40;
+
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class BaseTest {
+
+    public static void setup(WebDriver driver, String source) {
```

Comment:
1) You can add  BeforeEach annotation. 
Also after method can be moved to this class(where you close driver)
2) Method shouldn't be static. 
3) It shouldn't has method parameters:  WebDriver driver, String source